src/Routes/api_routes.php

```diff
@@ -190,133 +176,122 @@
 // Projects list
 $api->add('api-projects', new Route(
     '/projects',
-    array('_controller' => 'Goteo\Controller\Api\ProjectsApiController::projectsAction',
-        )
+    array('_controller' => 'Goteo\Controller\Api\ProjectsApiController::projectsAction',)
 ));
 
 // Projects by sdg
-$api->add('api-projects', new Route(
+$api->add('api-projects-by-sdg', new Route(
```

Comment:
@davidbeig @javicarrillo the issue that broke the api projects search was here. An override of the 'api-projects' endpoint. With this change, the previous endpoint is up and working as before.